docs/Course Wiki/ECE Course Offerings/ECE350.md

```diff
@@ -32,7 +32,7 @@ Take this course to satisfy 3 out of 5 credit for EEs or if you have an avid int
 
 During the semester, the course assigns 13-14 written homeworks, due every week. These homework sets are generally not too difficult and will contain both calculations and derivations of expressions. Homeworks often contain bonus problems worth extra credit that are more difficult or annoying to do, but lead to some interesting insights in the material. It is recommended that students do them - the amount of extra credit received is usually enough to be able to drop one or two homeworks at the end of the semester. Students have cited that each week's homework can take very different amounts of time, from under an hour to upwards of five hours to complete. They can sometimes be surprisingly annoying. It is recommended to start early so you can go to office hours.
 
-The course will also have three midterms and a final. The midterms are generally not thought to be too difficult - more of your time will be spent on the homeworks. There are only a few practice exams - thus it is recommended to keep up with the homework every week. Professor Kudeki's notes have good practice questions with solutions in them, should the student find that they need more practice. Students are allowed to bring a cheatsheet (one 8.5x11 front and back), as well as cheatsheets from prior midterms. For example, for midterm 3 you would be allowed three sheets front and back.
+The course will also have three midterms and a final. The midterms are generally not thought to be too difficult - more of your time will be spent on the homeworks. There are only a few practice exams - thus it is recommended to keep up with the homework every week. Professor Kudeki's notes have good practice questions with solutions in them, should the student find that they need more practice. Students are allowed to bring a cheatsheet (one 8.5x11 front and back), as well as cheatsheets from prior midterms. For example, for midterm 3 you would be allowed three sheets front and back. You are highly recommended to reuse cheatsheets from prior midterms, but this is not enforced.
```

Comment:
see wording comment above for how to make the policy more clear, last sentence here contradicts "as well as cheatsheets from prior exams"